### ISSUE TITLE
New quests: BBQ

### ISSUE THREAD

**Issue by michalgwo** [streetcomplete/StreetComplete#4832]
### General
We can add 3 quests for [amenity=bbq](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dbbq)

1. [covered](https://wiki.openstreetmap.org/wiki/Key:covered)
- Question asked: Is this place protected from rain? 
- Answers: yes/no

2. [fuel](https://wiki.openstreetmap.org/wiki/Key:fuel)
- Question asked: What is this grill powered by?
- Answers: wood/electric/charcoal

3. [wood_provided](https://wiki.openstreetmap.org/wiki/Key:wood_provided) (in case of fuel=wood)
- Question asked: Is wood provided by the operator of this bbq?
- Answers: yes/no

### Checklist
Checklist for quest suggestions (see [guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md)):
- [x] 🚧 To be added tag is established and has a useful purpose
- [x] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
- [x] 🐿️ Easily answerable by any pedestrian from the outside but a survey is necessary
- [x] 💤 Not an overwhelming percentage of quests have the same answer (No spam)
- [x] 🕓 Applies to a reasonable number of map data (Worth the effort)

**Comment by mnalis** [comment]
And they could have `Other answers` - `It is not a bbq, but a firepit`, replacing `amenity=bbq` with [leisure=firepit](https://wiki.openstreetmap.org/wiki/Tag:leisure=firepit) .

**Comment by letypequividelespoubelles** [comment]
And you could add a fourth (yes/no) question : Is there a [grate](https://wiki.openstreetmap.org/wiki/Key%3Agrate) ?

**Comment by arrival-spring** [comment]
The BBQ page says that fuel may typically not be provided, but the fuel page doesn't say what to tag if no fuel is provided.

The value no has been used, but only 92 times so is hardly established.

**Comment by matkoniecz** [comment]
How many public BBQ are covered? 1% of all? less? more?

**Comment by michalgwo** [comment]
> The BBQ page says that fuel may typically not be provided, but the fuel page doesn't say what to tag if no fuel is provided.
> 
> The value no has been used, but only 92 times so is hardly established.

From my understanding, fuel here means how the bbq is powered, not what is provided.

> How many public BBQ are covered? 1% of all? less? more? (note that it is distinct from "how many bbg with `covered` have `yes` value" as covered ones are more likely to be tagged with `covered` key)

I don't know the statistics, but I don't think its uncommon, I've seen it a couple of times. Photo from my town below.

![2022-05-28](https://user-images.githubusercontent.com/114614618/220603374-998059f0-6a2f-407c-8ade-ad6164218794.jpg)

**Comment by andrewharvey** [comment]
Could also ask for `capacity=*`, ie. How many cooking surfaces are available at this BBQ? I commonly encounter 1, 2 sometimes more.

**Comment by westnordost** [comment]
1 and 2 seem useful.

**Comment by matkoniecz** [comment]
barbecue cover quest part is covered by #4905

@roptat and others: is anyone interested in implementing

>Question asked: What is this grill powered by?
>Answers: wood/electric/charcoal

It seems to be friendly one for starting with StreetComplete coding ( https://github.com/streetcomplete/StreetComplete/blob/master/CONTRIBUTING_A_NEW_QUEST.md may help )

**Comment by qugebert** [comment]
> barbecue cover quest part is covered by #4905
> 
> @roptat and others: is anyone interested in implementing
> 
> > Question asked: What is this grill powered by?
> > Answers: wood/electric/charcoal
> 
> It seems to be friendly one for starting with StreetComplete coding ( https://github.com/streetcomplete/StreetComplete/blob/master/CONTRIBUTING_A_NEW_QUEST.md may help )

Here's my attempt of implementing this quest: https://github.com/streetcomplete/StreetComplete/pull/5211
Please tell me, if i made any mistakes, but it worked fine when i tested it.

**Comment by derfasthirnlosenick** [comment]
I'd also put a "do you have to pay for this BBQ?" At least for certain countries.

At least in Australia, coin operated BBQs used to be a thing.

**Comment by andrewharvey** [comment]
> I'd also put a "do you have to pay for this BBQ?" At least for certain countries.
>
> At least in Australia, coin operated BBQs used to be a thing.

I'm all for asking the question as more complete data is better, but I've never encountered one that's paid before.

Asking, "Is this BBQ still in working order?" would be a good idea given many break down and are left unpaired.

**Comment by derfasthirnlosenick** [comment]
> I'm all for asking the question as more complete data is better, but I've never encountered one that's paid before.

I'd country-restrict the quest anyway - never seen one outside Australia (where they are *everywhere*), though. Google says they might 3xidt in New Zealand and potentially also USA and Canada.

**Comment by qugebert** [comment]
Ok, i see this is still open. so to summarize:
We had various quests requested here, two of them are already implemented

- [x] [Is this covered from rain?](https://github.com/streetcomplete/StreetComplete/issues/4832#issue-1592301362)  -> Done with #4905 
- [x] [What is this grill powered by?](https://github.com/streetcomplete/StreetComplete/issues/4832#issue-1592301362)  -> Done with #5211 
- [ ] [Is wood provided by the operator of this bbq?](https://github.com/streetcomplete/StreetComplete/issues/4832#issue-1592301362)  -> Not sure, if this is really common / useful
- [ ] [Is there a grate?](https://github.com/streetcomplete/StreetComplete/issues/4832#issuecomment-1439486613) -> I think this could make sense and if we have this quest, then we should apply this to firepits too.
- [ ] [Ask for capacity](https://github.com/streetcomplete/StreetComplete/issues/4832#issuecomment-1448007301) -> There was no feedback to this request
- [ ] [do you have to pay for this BBQ?](https://github.com/streetcomplete/StreetComplete/issues/4832#issuecomment-1704911360) -> Not really common.

**Comment by mnalis** [comment]
> * [x]  [Is there a grate?](https://github.com/streetcomplete/StreetComplete/issues/4832#issuecomment-1439486613) -> There was no feedback to this request

This one has been covered by `other answer` here: PR https://github.com/streetcomplete/StreetComplete/pull/5371

> * [ ]  [Is wood provided by the operator of this bbq?](https://github.com/streetcomplete/StreetComplete/issues/4832#issue-1592301362)  -> Not sure, if this is really common / useful

Dunno, haven't really seen those in Croatia nor neighboring countries. If it is common is some countries (it would need research in which ones first), it should be restricted to such countries, and made dependent on `fuel=wood` answer on existing quest.

> * [ ]  [Ask for capacity](https://github.com/streetcomplete/StreetComplete/issues/4832#issuecomment-1448007301) -> There was no feedback to this request

I don't see much use of that (also, is 1 big surface worse than 2 small ones? Would we tag `width`/`length`? It's not worth it, IMHO). 

> * [ ]  [do you have to pay for this BBQ?](https://github.com/streetcomplete/StreetComplete/issues/4832#issuecomment-1704911360) -> Not really common.

I agree. Perhaps if some county has them commonly, it might be limited to such countries, but unless someone comes up with such research, I wouldn't bother.

In short, I would think this can be closed as completed unless someone steps forward with research for those unimplemented quests.

**Comment by matkoniecz** [comment]
closing, but feel free to file new issues (using new quest issue template) if any of remaining quests is viable

(one quest per issue is best)
